src/RoomsSDKAdapter.js

```diff
@@ -159,6 +161,25 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
     return this.getRoomObservables[ID];
   }
 
+  /**
+   * Returns an observable that emits room data of the recently created room
+   * Observable will complete after one emission.
+   *
+   * @param {Room} room  Information about the room to create.
+   * @returns {external:Observable.<Room>} Observable stream that emits room data.
+   * @memberof RoomsAdapter
+   */
+  createRoom(room) {
+    logger.debug('ROOM', room, 'createRoom()', ['called with', {room}]);
+
+    return defer(() => this.datasource.rooms.create(room)).pipe(
+      catchError((err) => {
+        logger.error('ROOM', room, 'createRoom()', `Unable to create a room with "${room.title}"`, err);
```

Comment:
maybe not just the title is relevant in case of an error, and if the room.title is "test", the error will read "Unable to create a room with test", which is not very clear. I would suggest changing to:
```js
logger.error('ROOM', undefined, 'createRoom()', ['Unable to create room', {room}], err);